flexmeasures/api/v1_3/routes.py

```diff
@@ -120,6 +121,8 @@ def post_udi_event():
                     "datetime": "2015-06-02T16:00:00+00:00"
                 }
             ],
+            "soc_min": 10,
```

Comment:
I might misunderstand the two ways that `value` can be sent in here (one as a measurement, one as a target, right?)

It is kind of confusing what the "event" is, especially when both types of `value` are being sent at once. 

Anyway ― Isn't it cleaner if we require these SOC constraints to be sent in within the `targets` dict?

Reply:
The `value` in the top level of the JSON denotes an SOC measurement (used as the starting value for computing the schedule). The `value` in the list of `targets` denotes a wish list of future values. Note that `targets` is a list rather than a dict, so `targets` can contain multiple values.

The `soc_min` and `soc_max` are not part of the "targets" wish list, but rather fixed constraints.

Reply:
Okay it makes sense to leave it where it is.

Maybe we should rename the measurement-value to `start-value`? Of course, if this endpoint is used to only send that measurement, it does not make sense. For our future API design, I have this feeling that this should be two very clearly-defined endpoints.